devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -57,10 +57,9 @@
   with_items:
       - securitas.service
 
-- name: Start and enable the securitas services
-  service:
-      name: "{{ item }}"
-      state: started
-      enabled: yes
-  with_items:
-      - securitas
\ No newline at end of file
+- name: Reload systemd so it learns about securitas
```

Comment:
The name does not describe what the task does, the task also starts the daemon (and that's the main thing, I would say)